package.json

```diff
@@ -38,6 +38,9 @@
     "react-dom": ">=16",
     "react-redux": "7"
   },
+  "react-native": {
+    "https": false
+  },
```

Comment:
@gre We needed this in wallet-btc to be compatible with react-native.
Hope it's ok to have it in live-common.